package-lock.json

```diff
@@ -2013,12 +2180,37 @@
             "pretty-hrtime": "^1.0.0",
             "replace-homedir": "^1.0.0",
             "semver-greatest-satisfied-range": "^1.1.0",
-            "v8flags": "^3.2.0"
-          },
-          "dependencies": {
-            "yargs": {
-              "version": "13.3.2"
-            }
+            "v8flags": "^3.2.0",
+            "yargs": "^7.1.0"
```

Comment:
We may have to come up with a better solution to this issue, but your push has reverted the security fix, of upgrading yargs.